eng/swagger_to_sdk_config.json

```diff
@@ -9,5 +9,8 @@
     "initScript": {
       "path": "pwsh ./eng/scripts/Automation-Sdk-Init.ps1"
     }
+  },
+  "packageOptions": {
+    "packageFolderFromFileSearch": false
```

Comment:
@dw511214992 can you better help me understand what this setting does? Is it something we want to be set for all packages within this automation?